pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -1796,21 +1799,37 @@ func getSharedDatastoresInClusters(ctx context.Context, clusterMorefs []string,
 func (c *K8sOrchestrator) StartZonesInformer(ctx context.Context,
 	restClientConfig *restclient.Config, namespace string) error {
 	log := logger.GetLogger(ctx)
+	var startErr error
```

Comment:
All instances of StartZonesInformer is called with restClientConfig=nil, can we update the call signature to not specify it in the first place?